.gitlab/functional_test/workload_checks.yml

```diff
@@ -3,7 +3,7 @@ single-machine-performance-workload-checks:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/docker_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
   tags: ["runner:docker"]
   rules:
-    !reference [.on_deploy_nightly_repo_branch_a7]
+    !reference [.on_scheduled_main]
```

Comment:
The intent of this configuration was to scope running Workload Checks on nightly release images based on `main`. My rough understanding of Agent CI was that developers could also create "nightly" images based on other feature branches, and Single Machine Performance wanted to exclude nightly images that were not based on `main`.

Reply:
If this change achieves that intent, I'm happy to approve (or to discuss in person, because I'm also in Paris for the Agent Summit).